regression/scripts/Vehicles/Concorde.py

```diff
@@ -53,52 +53,52 @@ def vehicle_setup(source_ratio=1.):
     wing = SUAVE.Components.Wings.Wing()
```

Comment:
This has openvsp meshing associated with it. We need to make sure when it's run the CFD still converges if that's a desired quality. @timdmacdo  what do you think?